lib/src/interop/storage_interop.dart

```diff
@@ -76,7 +76,6 @@ class FullMetadataJsImpl extends UploadMetadataJsImpl {
 @anonymous
 class UploadMetadataJsImpl extends SettableMetadataJsImpl {
   external String get md5Hash;
-  external void set md5Hash(String s);
```

Comment:
@kevmoo 
About this setter. I added it here because of this table: https://firebase.google.com/docs/storage/web/file-metadata#file_metadata_properties
They say here for write:
md5Hash | string | YES on upload, NO on updateMetadata